PackageInfo.g

```diff
@@ -93,7 +93,7 @@ PackageDoc := rec(
 
 Dependencies := rec(
   GAP := ">= 4.5",
-  NeededOtherPackages := [ [ "GAPDoc", ">= 1.5" ] ],
+  NeededOtherPackages := [ [ "GAPDoc", ">= 1.6.1" ] ],
```

Comment:
GAPDoc 1.6.1 has not yet been released, so I don't think we should merge this before.